presto-main/src/main/java/com/facebook/presto/sql/rewrite/ShowQueriesRewrite.java

```diff
@@ -219,27 +218,11 @@ protected Node visitShowGrants(ShowGrants showGrant, Void context)
                 throw new SemanticException(CATALOG_NOT_SPECIFIED, showGrant, "Catalog must be specified when session catalog is not set");
             }
 
-            Optional<String> optionalTableName = Optional.empty();
-            if (tableName.isPresent()) {
-                optionalTableName = Optional.of(tableName.get().getSuffix());
-            }
-
-            accessControl.checkCanShowGrants(
-                    session.getRequiredTransactionId(),
-                    session.getIdentity(),
-                    new QualifiedTablePrefix(catalogName, session.getSchema(), optionalTableName));
-
             return simpleQuery(
-                    selectList(
-                            aliasedName("grantee", "Grantee"),
-                            aliasedName("table_catalog", "Catalog"),
-                            aliasedName("table_schema", "Schema"),
-                            aliasedName("table_name", "Table"),
-                            aliasedName("privilege_type", "Privilege"),
-                            aliasedName("is_grantable", "Grantable")),
+                    selectList(new AllColumns()),
```

Comment:
Is there a need to remove the aliases? The output produced by the original code is more readable, in my opinion.